elf/module.go

```diff
@@ -467,8 +472,14 @@ func (b *Module) closeMaps() error {
 //
 // It doesn't detach BPF programs from cgroups or sockets because they're
 // considered resources the user controls.
+// It also doesn't unpin pinned maps. Use CloseExt and set Unpin to do this.
 func (b *Module) Close() error {
-	if err := b.closeMaps(); err != nil {
+	return b.CloseExt(nil)
+}
+
+// CloseExt takes a map "elf section -> CloseOptions"
```

Comment:
it should be documented somewhere whether it unpins a map if it is not mentioned in the map. What's the default value of the Unpin bool?